crates/configuration/src/relaychain.rs

```diff
@@ -101,7 +107,7 @@ impl Default for RelaychainConfigBuilder<Initial> {
             config: RelaychainConfig {
                 chain: "default"
                     .try_into()
-                    .expect("'default' overriding should be ensured by typestate. this is a bug, please report it: https://github.com/paritytech/zombienet-sdk/issues"),
+                    .unwrap_or_else(|_| panic!("{} {}", DEFAULT_TYPESTATE, THIS_IS_A_BUG)),
```

Comment:
We should disable this lint from clippy and revert to using expect with &format: https://github.com/rust-lang/rust-clippy/issues/2928